metric/system/process/process.go

```diff
@@ -27,6 +27,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/joeshaw/multierror"
```

Comment:
https://pkg.go.dev/errors supports wrapping multiple errors now, we don't need a new dependency for this. See https://pkg.go.dev/errors#Join

Reply:
Got it! Thanks for pointing this out.